roles/oso_host_monitoring/templates/oso-rhel7-host-monitoring.service.j2

```diff
@@ -62,6 +62,8 @@ ExecStart=/usr/bin/docker run --name {{ osohm_host_monitoring }}
            -v /etc/origin/master/master.etcd-client.crt:/etc/origin/master/master.etcd-client.crt \
            -v /etc/origin/master/master.etcd-client.key:/etc/origin/master/master.etcd-client.key \
            -v /etc/origin/master/master-config.yaml:/etc/origin/master/master-config.yaml \
+{% elif hostvars[inventory_hostname]['ec2_tag_host-type'] == 'node' %}
```

Comment:
We should start using the oo_ vars as they'll be cross cloud.

So, `oo_hosttype`.